### ISSUE TITLE
Terminal suggest: No suggestion obscures line

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#236338]
Repro:

1. Open pwsh
2. Type `./`, 🐛 no suggestions obscures the line

![Image](https://github.com/user-attachments/assets/b766b93b-4332-4cdf-a9f9-98de18e0ad2f)

It should show on the above line like the suggest widget.

**Comment by Tyriar** [comment]
Also it should provide suggestions here, not sure how to trigger it to show no suggestions but it happened a few times. Likely related to process reconnect/reloading window.

**Comment by meganrogge** [comment]
Yeah, if suggestions aren't showing up, it's almost certainly because of https://github.com/microsoft/vscode/issues/234672

**Comment by meganrogge** [comment]
Shows below for the editor's suggest widget 

<img width="377" alt="Image" src="https://github.com/user-attachments/assets/39db4af1-6ab4-4f6e-9ada-eebdf18b854f" />

**Comment by meganrogge** [comment]
It uses padding-left

https://github.com/microsoft/vscode/blob/68410e1431f9849547b67d0b38dbc76cdb4aa3c8/src/vs/editor/contrib/suggest/browser/media/suggest.css#L83-L87

**Comment by meganrogge** [comment]
With the fix for https://github.com/microsoft/vscode/issues/236368, you'll see this much less. I looked into it for a while and still not sure the cause, so sidelining it for now.

**Comment by Tyriar** [comment]
I saw this just now and it doesn't move when the cursor moves, so it totally obscures what I'm typing:

Repro:

1. Paste `until ! ; do :; done` into bash
2. Move cursor before the first `;`
3. Type `./scripts/test`
4. Escape to hide suggest
5. ctrl+space to request completions

<img width="397" alt="Image" src="https://github.com/user-attachments/assets/b4db298e-60b5-4cbe-bd1b-1936faa3a4cf" />

After typing more see cursor on the right side of suggest popup

<img width="521" alt="Image" src="https://github.com/user-attachments/assets/7cbbb8e8-dc70-4ba4-aeca-223e1ccd6599" />

**Comment by meganrogge** [comment]
For this case, `this._list.contentHeight` is zero, which results in this for the element height and top values

<img width="405" alt="Image" src="https://github.com/user-attachments/assets/0818ad12-4a4c-4e58-8ee0-d41b8f9ee199" />

**Comment by meganrogge** [comment]
replacing `this._lost.contentHeight` with `Math.max(this._list.contentHeight, 20)` fixes that

<img width="336" alt="Image" src="https://github.com/user-attachments/assets/a229a71e-08c3-4b44-a7cb-e35cdb2c0ad8" />

**Comment by meganrogge** [comment]
Looks like the list element is hidden and message element gets shown, so may need to factor in that height

**Comment by meganrogge** [comment]
Yep, that works 

<img width="666" alt="Image" src="https://github.com/user-attachments/assets/7cea812d-6c8a-4afd-adc1-6ca7d32fd921" />
